app.js

```diff
@@ -52,4 +52,254 @@ document.addEventListener('DOMContentLoaded', () => {
       page.className = '';
     }
   });
+
+  // Create Dynamic Page
+  const grand = document.querySelector('.show-inactive');
+  const content = document.querySelector('.featured-item');
```

Comment:
- your modal popup window does not look like the provided figma design. Kindly fix the styling to match the provided design.

Your design
![image](https://user-images.githubusercontent.com/81932478/181622886-df413d60-13b6-4b67-b108-d22b8dcbcec6.png)

FIgma design
![image](https://user-images.githubusercontent.com/81932478/181623135-d3ca4c01-794d-464e-9c18-6aa7a2eb2f30.png)

- [ ] The bottom buttons are not visible
- [ ] The color of the text is also not as the one in the provided design
- [ ] The top heading is also not visible
- [ ] The languages at the top are also not visible.
- [ ] The close button is anot accessible as well

Kindly fix these issues and follow the provided layout, design, fonts and colors.